src/mono/wasm/runtime-test.js

```diff
@@ -119,6 +119,7 @@ setenv = {};
 runtime_args = [];
 enable_gc = true;
 enable_zoneinfo = false;
+working_dir = "net5.0-Browser-debug";
```

Comment:
Instead of hardcoding this here, we should pass this in as an argument to `--working-dir` that you are adding.